src/Plotly.NET/Plotly.NET.fsproj

```diff
@@ -1,6 +1,8 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <TargetFrameworks>netstandard2.0; net5.0</TargetFrameworks>
+    <SignAssembly>true</SignAssembly>
+    <AssemblyOriginatorKeyFile>Open.snk</AssemblyOriginatorKeyFile>
```

Comment:
You will need to [create your own public-private key](https://docs.microsoft.com/en-us/dotnet/standard/assembly/create-public-private-key-pair#create-a-key-pair).

Reply:
You can create it yourself without the passphrase and push it to the repo.